content/content-strategy/index.yml

```diff
@@ -1,6 +1,6 @@
 pagetitle: Content Strategy Guide
 weight: 60
-description: A content strategy sets the foundation for the creation, structure and governance of content. It can be used for any new services, content redesigns or big changes such as Machinery of Government changes.
+description: Use our content strategy guide to create clear information structures for your websites. Learn how to set up governance for content creation in your workplace. Refer to this guide for new services, content redesigns, or Machinery of Government changes.
```

Comment:
I think we need the `or big changes such as ... ` to make sense - this helps you deal with content impacts of the changes, not a MOG itself. But I can't find a way of avoiding reusing `changes`. Thoughts @annekemyers ?